### PR TITLE
Add concept blurb: booleans, strings

### PR DESCRIPTION
**Pull request by mirkoperillo**
Blurb proposal for concepts:
* booleans
* strings

### CODE REVIEW

File: concepts/booleans/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Learn about boolean operations.",
```
Comment (ErikSchierboom) [inline concepts/booleans/.meta/config.json:2]: Concept blurbs should be more like descriptions, see https://github.com/exercism/csharp/blob/main/concepts/booleans/.meta/config.json#L2 for an example.

File: concepts/strings/.meta/config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Learn about string formatting.",
```
Comment (ErikSchierboom) [inline concepts/strings/.meta/config.json:2]: See above comment.

**Review by ErikSchierboom — COMMENTED** [review]
Thanks! I've left two nits.

**Review by ErikSchierboom — APPROVED** [review]

File: concepts/strings/.meta/config.json (hunk 2 above)
Comment (ErikSchierboom) [reply concepts/strings/.meta/config.json]: I've approved with an update to this blurb in mind :)

File: concepts/booleans/.meta/config.json (hunk 1 above)
Comment (mirkoperillo) [reply concepts/booleans/.meta/config.json]: I tried to improve it

File: concepts/strings/.meta/config.json (hunk 2 above)
Comment (mirkoperillo) [reply concepts/strings/.meta/config.json]: I tried to improve it

**Comment by ErikSchierboom** [comment]
Perfect, thanks!

**Pull request merged**